src/Servers/IIS/IIS/test/Common.FunctionalTests/ShutdownTests.cs

```diff
@@ -347,11 +415,22 @@ public async Task OutOfProcessToInProcessHostingModelSwitchWorks()
             await deploymentResult.HttpClient.RetryRequestAsync("/HelloWorld", r => r.Headers.Server.ToString().StartsWith("Microsoft"));
         }
 
-        [ConditionalTheory]
-        [InlineData(HostingModel.InProcess)]
-        [InlineData(HostingModel.OutOfProcess)]
+        [ConditionalFact]
+        [QuarantinedTest("https://github.com/dotnet/aspnetcore-internal/issues/1794")]
+        public async Task ConfigurationTouchedStress_InProcess()
+        {
+            await ConfigurationTouchedStress(HostingModel.InProcess);
+        }
+
+        [ConditionalFact]
+        [RequiresNewShim]
         [QuarantinedTest("https://github.com/dotnet/aspnetcore-internal/issues/1794")]
```

Comment:
This test has only failed once in the last 30 days, and it was InProcess. You can unquarantined OutOfProcess.